src/rustdoc-internals.md

```diff
@@ -176,8 +176,9 @@ that deal with it. Since we have the full `compiletest` suite at hand, there's
 a set of tests in `src/test/rustdoc` that make sure the final HTML is what we
 expect in various situations. These tests also use a supplementary script,
 `src/etc/htmldocck.py`, that allows it to look through the final HTML using
-XPath notation to get a precise look at the output. The full description of all
-the commands available to rustdoc tests is in `htmldocck.py`.
+XPath notation to get a precise look at the output. [The full description] of
+all the commands available to rustdoc tests (e.g. [`@has`] and [`@matches`]) is
+in `htmldocck.py`.
```

Comment:
nit: I would make `htmldocck.py` a link instead of `The full description`.

Reply:
I was about to comment I should do that, I'll fix it !